requirements/edx/custom.txt

```diff
@@ -20,14 +20,14 @@ git+https://github.com/edx-solutions/xblock-group-project.git@1.0.0#egg=xblock-g
 -e git+https://github.com/mckinseyacademy/xblock-group-project-v2.git@0.12.14#egg=xblock-group-project-v2==0.12.14
 -e git+https://github.com/mckinseyacademy/xblock-virtualreality.git@v0.1.5#egg=xblock-virtualreality==0.1.5
 
--e git+https://github.com/edx/edx-notifications.git@3.0.0#egg=edx-notifications==3.0.0
-git+https://github.com/edx-solutions/gradebook-edx-platform-extensions.git@3.0.3#egg=gradebook-edx-platform-extensions==3.0.3
-git+https://github.com/edx-solutions/mobileapps-edx-platform-extensions.git@3.0.0#egg=mobileapps-edx-platform-extensions==3.0.0
-git+https://github.com/edx-solutions/discussion-edx-platform-extensions.git@3.0.1#egg=discussion-edx-platform-extensions==3.0.1
-git+https://github.com/edx-solutions/organizations-edx-platform-extensions.git@v3.0.1#egg=organizations-edx-platform-extensions==3.0.1
-git+https://github.com/edx-solutions/course-edx-platform-extensions.git@v4.0.0#egg=course-edx-platform-extensions==4.0.0
-git+https://github.com/edx-solutions/projects-edx-platform-extensions.git@v4.0.0#egg=projects-edx-platform-extensions==4.0.0
-git+https://github.com/edx-solutions/api-integration.git@v5.0.5#egg=api-integration==5.0.5
+-e git+https://github.com/mckinseyacademy/edx-notifications.git@3.0.0#egg=edx-notifications==3.0.0
+git+https://github.com/mckinseyacademy/gradebook-edx-platform-extensions.git@3.0.3#egg=gradebook-edx-platform-extensions==3.0.3
+git+https://github.com/mckinseyacademy/mobileapps-edx-platform-extensions.git@3.0.0#egg=mobileapps-edx-platform-extensions==3.0.0
+git+https://github.com/mckinseyacademy/discussion-edx-platform-extensions.git@3.0.1#egg=discussion-edx-platform-extensions==3.0.1
+git+https://github.com/mckinseyacademy/organizations-edx-platform-extensions.git@v3.0.1#egg=organizations-edx-platform-extensions==3.0.1
+git+https://github.com/mckinseyacademy/course-edx-platform-extensions.git@v4.0.0#egg=course-edx-platform-extensions==4.0.0
+git+https://github.com/mckinseyacademy/projects-edx-platform-extensions.git@v4.0.0#egg=projects-edx-platform-extensions==4.0.0
+git+https://github.com/mckinseyacademy/api-integration.git@v2.0.0#egg=api-integration==2.0.0
```

Comment:
Can you check if version is correct for api-integeration? i.e v5.0.5

Reply:
This tag newly created at MCKA repo which is a mirror of 5.0.5 of edx-solutiions